src/groupeddataframe/splitapplycombine.jl

```diff
@@ -818,9 +820,12 @@ function _combine_with_first(first::Union{NamedTuple, DataFrameRow, AbstractData
     let eltys=eltys, n=n # Workaround for julia#15276
         initialcols = ntuple(i -> Tables.allocatecolumn(eltys[i], n), _ncol(first))
     end
-    outcols = _combine_with_first!(first, initialcols, idx, 1, 1, f, gd, incols,
-                                   tuple(propertynames(first)...))
-    idx, outcols, propertynames(first)
+    targetcolnames = tuple(propertynames(first)...)
+    outcols, finalcolnames = first isa Union{AbstractDataFrame,
+                              NamedTuple{<:Any, <:Tuple{Vararg{AbstractVector}}}} ?
```

Comment:
Weird indentation. Turn this into an `if`?

Reply:
I was not sure what would be best. Changed to `if`